src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -160,28 +165,36 @@ class CommunityAPIActionPlanEventService(
 
   override fun onApplicationEvent(event: ActionPlanEvent) {
     when (event.type) {
-      ActionPlanEventType.SUBMITTED,
-      -> {
+      ActionPlanEventType.SUBMITTED -> {
+        val url = UriComponentsBuilder.fromHttpUrl(interventionsUIBaseURL)
+          .path(interventionsUISubmittedActionPlanLocation)
+          .buildAndExpand(event.actionPlan.referral.id)
+          .toString()
+
+        postNotificationRequest(event, url, event.actionPlan.submittedAt!!)
+      }
+      ActionPlanEventType.APPROVED -> {
         val url = UriComponentsBuilder.fromHttpUrl(interventionsUIBaseURL)
-          .path(interventionsUIActionPlanLocation)
+          .path(interventionsUIApprovedActionPlanLocation)
           .buildAndExpand(event.actionPlan.referral.id)
           .toString()
 
-        postNotificationRequest(event, url)
+        postNotificationRequest(event, url, event.actionPlan.approvedAt!!)
       }
     }
   }
 
-  private fun postNotificationRequest(event: ActionPlanEvent, url: String) {
+  private fun postNotificationRequest(event: ActionPlanEvent, url: String, eventTime: OffsetDateTime) {
 
     val referral = event.actionPlan.referral
+    val status = camelCase(event.type.name)
```

Comment:
i would prefer an explicit `when` block here. i think it's clearer what it's doing, and it decouples the event type enum name from the delius status.

```
val status = when(event.type) {
  APPROVED -> "Approved"
  SUBMITTED -> "Submitted"
}

Reply:
Will do